coderadar-test/src/test/java/io/reflectoring/coderadar/rest/useradministration/permissions/RemoveUserFromProjectControllerIntegrationTest.java

```diff
@@ -0,0 +1,58 @@
+package io.reflectoring.coderadar.rest.useradministration.permissions;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+
+import io.reflectoring.coderadar.graph.projectadministration.domain.ProjectEntity;
+import io.reflectoring.coderadar.graph.projectadministration.project.repository.ProjectRepository;
+import io.reflectoring.coderadar.graph.useradministration.domain.UserEntity;
+import io.reflectoring.coderadar.graph.useradministration.repository.UserRepository;
+import io.reflectoring.coderadar.rest.ControllerTestTemplate;
+import io.reflectoring.coderadar.useradministration.service.security.PasswordUtil;
+import java.util.Collections;
+import java.util.Date;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+class RemoveUserFromProjectControllerIntegrationTest extends ControllerTestTemplate {
+
+  @Autowired private UserRepository userRepository;
+
+  @Autowired private ProjectRepository projectRepository;
+
+  private ProjectEntity testProject;
+  private UserEntity testUser;
+
+  @BeforeEach
+  void setUp() {
+    projectRepository.deleteAll();
+    testProject = new ProjectEntity();
+    testProject.setVcsUrl("https://valid.url");
+    testProject.setName("project");
+    testProject.setVcsStart(new Date());
+    testProject.setVcsOnline(true);
+    testProject.setVcsPassword("testPassword");
+    testProject.setVcsUsername("testUser");
+    projectRepository.save(testProject);
+
+    userRepository.deleteAll();
```

Comment:
Zu Beginn eines jeden Tests sollte die Datenbank leer sein, Aufräumarbeiten sollten entsprechend in der `@AfterEach`-Methode geschehen.

Reply:
In IntegrationTestTemplate ausgelagert.